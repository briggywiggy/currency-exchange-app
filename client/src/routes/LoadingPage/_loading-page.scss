.loading-page {
    align-items: center;
    display: flex;
    justify-content: center;
}

.loading-page__image {
    height: 20rem;
    width: auto;
}