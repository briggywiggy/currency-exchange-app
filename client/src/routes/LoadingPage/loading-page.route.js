import React from 'react';

const LoadingPage = () => (
    <div className="loading-page page">
        <img className="loading-page__image" src="/images/loader_alt.gif" />
    </div>
)

export default LoadingPage;