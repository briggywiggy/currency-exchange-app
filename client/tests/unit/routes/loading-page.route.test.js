import React from 'react';
import { shallow } from 'enzyme';
import LoadingPage from '../../../src/routes/LoadingPage/loading-page.route';

test('should render LoadingPage correctly', () => {
    const wrapper = shallow(<LoadingPage />);
    expect(wrapper).toMatchSnapshot();
})