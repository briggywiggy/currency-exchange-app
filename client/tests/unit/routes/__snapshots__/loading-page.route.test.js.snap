// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardPage correctly 1`] = `undefined`;

exports[`should render LoadingPage correctly 1`] = `
<div
  className="loading-page page"
>
  <img
    className="loading-page__image"
    src="/images/loader_alt.gif"
  />
</div>
`;
